guides/common/modules/proc_creating-a-custom-ssl-certificate.adoc

```diff
@@ -36,29 +36,19 @@ If you already have a private key for this {ProductName}, skip this step.
 [ req ]
 req_extensions = v3_req
 distinguished_name = req_distinguished_name
-x509_extensions = usr_cert
 prompt = no
 
 [ req_distinguished_name ]
-CN = _{context}.example.com_
+commonName = _{smartproxy-example-com}_
```

Comment:
If this file is re-used for both Foreman Server and Foreman SmartProxy, then `{context}.example.com` is correct.

Reply:
This doesn't render correctly for https://theforeman-foreman-documentation-preview-pr-2832.surge.sh/nightly/Installing_Server/index-satellite.html#creating-a-custom-ssl-certificate_satellite

Reply:
I think we should not overwrite the context but instead "wrap" the module in the assembly with a new attributes such as "target" or "ssl-common-name" or similar. Then we have full control and only change what we actually want to change without affecting other instances of "{context}".

Reply:
@maximiliankolb sounds good to me! @Lennonka should I do this instead?

Reply:
Yes, please.

Reply:
@Lennonka @maximiliankolb done. Does this look okay now?